scripts/vagrant/install.sh

```diff
@@ -4,20 +4,33 @@ cd "$(dirname $0)/.."
 
 # Kick start Hitchwiki installation
 
-echo "------------------------------------------------------------"
-echo ""
+sep="\n------------------------------------------------------------\n"
+
+echo -e $sep
 echo "   o  o o-O-o o-O-o   o-o o  o o       o o-O-o o  o o-O-o"
 echo "   |  |   |     |    /    |  | |       |   |   | /    |  "
 echo "   O--O   |     |   O     O--O o   o   o   |   OO     |  "
 echo "   |  |   |     |    \    |  |  \ / \ /    |   | \    |  "
 echo "   o  o o-O-o   o     o-o o  o   o   o   o-O-o o  o o-O-o"
 echo ""
 echo "     The Hitchhiker's Guide to Hitchhiking the World"
-echo ""
-echo "------------------------------------------------------------"
-echo ""
+echo -e $sep
+
+echo "Checking system ..."
```

Comment:
We'll have so many different systems where this file might be run, that it's better to just have these steps listed at INSTALL.md's "Prerequisites" section. I just added Ansible there.

E.g. I'm running MacOS and this just throws a bunch of errors for me. :-)

Reply:
just pushed a commit fixing the bash issue